app/javascript/components/nav/component.jsx

```diff
@@ -76,6 +78,7 @@ const Nav = () => {
           jewelCount={jewelCount}
           username={username}
           closeDrawer={handleToggleDrawer(false)}
+          show={menuEntry.show}
```

Comment:
Instead of hiding the component with CSS, lets skip rendering the component when !menuEntry.show.

Reply:
Done!